composer.json

```diff
@@ -1,12 +1,16 @@
 {
-    "name": "bramus/ansi-php",
-    "description": "ANSI Control Functions and ANSI Control Sequences (Colors, Erasing, etc.) for PHP CLI Apps",
+    "name": "nahkampf/ansi-php",
+    "description": "ANSI Control Functions and ANSI Control Sequences (Colors, Erasing, etc.) for PHP CLI Apps. A fork of bramus/ansi-php.",
```

Comment:
To accept this PR into this repo, this change needs to be removed as it’s changing the package name.